src/ConfigGen/Program.cs

```diff
@@ -254,7 +254,7 @@ static string parseType(string type)
         static string createFileHeader(string branch)
         {
             return
-@"// *** Auto-generated from librdkafka " + branch + @" *** - do not modify manually.
+@"// *** Auto-generated from librdkafka branch " + branch + @" *** - do not modify manually.

```

Comment:
took the word 'branch' out before you forked evidently. should have no diff here.

Reply:
removed the "Branch" text